server/src/main/java/org/elasticsearch/search/fetch/FetchPhase.java

```diff
@@ -72,8 +74,12 @@ public void execute(SearchContext context) {
 
         Profiler profiler = context.getProfilers() == null ? Profiler.NOOP : Profilers.startProfilingFetchPhase();
         SearchHits hits = null;
-        try {
-            hits = buildSearchHits(context, profiler);
+        try (XContentBuilder xContentValidator = new XContentBuilder(XContentType.JSON.xContent(), OutputStream.nullOutputStream())) {
```

Comment:
Question for better understanding: this validator builder is used to completely serialize the search hits once on the node before sending them across the wire and then thrown away? Is there an understanding of the extra work this involves? I see why validation is necessary for chunking but I want to make sure I understand the tradeoffs here.

Reply:
Actually I think #95673 means that I can remove this bit, good catch!